doc/api/test.md

```diff
@@ -219,6 +219,46 @@ test('a test that creates asynchronous activity', (t) => {
 });
 ```
 
+## Running tests from the command line
+
+The Node.js test runner can be invoked from the command line by passing the
+[`--test`][] flag. By default, Node.js will recursively search the current
+directory for test files:
```

Comment:
what constitutes a "test file"?

There is very much *not* a universal convention in the JS ecosystem, so although I see it's detailed below, I think there needs to be either a link to the lower section, or more exposition on what this means.